core/frontend/src/RealityDataSource.ts

```diff
@@ -221,6 +229,11 @@ export namespace RealityDataSource {
  * is invoked to produce the reality data source.
  * @alpha
  */
+
+// TODO extend this for GP3DT
```

Comment:
how applicable are these todos still?

Reply:
`RealityDataSourceGP3DTProvider` is still technically WIP so I reverted this to draft

Reply:
Thanks for reverting to draft, @eringram. Thanks @aruniverse for the feedback - we were unsure how fast we would try to push this through until we had some more discussions.

Reply:
@aruniverse Can this one be resolved now? I believe all these TODOs are taken care of and removed.